File: Java/백준/Bronze/21354. Äpplen och päron/Äpplen och päron.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        int a = in.nextInt()*7;
        int b = in.nextInt()*13;
        if(a==b){
            System.out.println("lika");
        }
        else if(a>b){
            System.out.println("Axel");
        }
        else{
            System.out.println("Petra");
        }
    }
}

File: C#/백준/Bronze/21354. Äpplen och päron/Äpplen och päron.cs
using System;

namespace q{
    class w{
        static void Main(){
            int[] a = Array.ConvertAll(Console.ReadLine().Split(),int.Parse);
            a[0]*=7; a[1]*=13;
            if(a[0]>a[1]){
                Console.WriteLine("Axel");
            }
            else if(a[0]<a[1]){
                Console.WriteLine("Petra");
            }
            else{
                Console.WriteLine("lika");
            }
        }
    }
}